models/opensea/ethereum/opensea_v1_ethereum_events.sql

```diff
@@ -155,6 +155,9 @@ SELECT DISTINCT
   'Buy' AS trade_category,
   wa.seller AS seller,
   wa.buyer AS buyer,
+  CASE WHEN buyer=agg.contract_address AND erct2.to IS NOT NULL THEN erct2.to
+    WHEN buyer=agg.contract_address AND erct3.to IS NOT NULL THEN erct3.to
+    ELSE buyer END AS buyer
```

Comment:
looks like ```buyer``` column is derived twice & then no ```,``` after

Reply:
Ah yeah thanks for the heads up, I addressed this in a new commit

Reply:
```buyer``` is still derived twice. i think you need to remove the old column, similar to other models here?